### DISCUSSION TITLE
cannot find package "." in: when using go generate command

### DISCUSSION THREAD

**Discussion by khanakia** [google/wire#289] [Q&A]
When i use `wire` command it generates the code but after that if i use the `go generate` command i am getting the following error

```
cannot find package "." in:
	/Volumes/D/www/go1/kookgo/vendor/github.com/google/wire/cmd/wire
wire_gen.go:3: running "go": exit status 1
```

**Comment by cglotr** [comment]
I face the same issue & I am able to fix it by specifying the packages like [this](https://github.com/arikama/koran-backend/blob/master/bashscripts/gogen.sh)
